build.gradle

```diff
@@ -108,6 +105,17 @@ subprojects { subProject ->
 
   if (!path.startsWith(":samples") && !path.startsWith(":misk-embedded-sample")) {
     apply plugin: 'com.vanniktech.maven.publish'
+
+    // Override signing plugin details that maven-publish will use.
+    signing {
```

Comment:
this breaks local publishing, so I guess I can put this part in the `hooks.gradle`?